### ISSUE TITLE
review undo function 

### ISSUE THREAD

**Issue by Slackman2015** [storyblok/storyblok#473]
**The feature would affect:** (check one with "x")   
- [x] *app.storyblok.com (CMS - Interface)*   
- [ ] *api.storyblok.com (CMS - Content Delivery API)*
- [ ] *mapi.storyblok.com (CMS - Management API)*
- [ ] *capi.storyblok.com (Commerce - API)*     
- [ ] *Commerce - Interface*   
- [ ] *Other* 

**Is your feature request related to a problem? Please describe.**
The in a previous ticket https://github.com/storyblok/storyblok/issues/138 requested and implemented feature needs a review as its implementation doesn't comply with the usual standard how the customers are used.

The problem is that it goes only one character back every time.

**Describe the solution you'd like**
A better solution would be that the undo function would work like the standard used in other applications for example that it undoes changes word by word or similar.

**Comment by abrambailey** [comment]
Yes, ironically this issue was created by my previous feature request (to have an undo button)... The undo button is good because it helps with undoing actions in the CMS interface, but the issue is the new functionality somehow overrides the general keyboard undo function. I think the keyboard should just be left to normal behavior for undoing text modifications and force people to use the UI button to undo storyblok related changes to blocks etc.

**Comment by abrambailey** [comment]
Also I would classify this as a bug @Slackman2015 rather than an enhancement... the undo keyboard function is technically broken and this is very frustrating. Would also love to see someone assigned to this.

**Comment by alvarosabu** [comment]
Hi @Slackman2015, we are closing this issue because this repository was used in the early days for support-related questions, but we've since transitioned to more effective channels for helping our users.

Please, refer to the following resources:

Storyblok Github Org: https://github.com/storyblok
Storyblok Docs: https://www.storyblok.com/docs
Official Support: https://support.storyblok.com
Join our Discord: https://www.storyblok.com/join-discord

Thanks for the understanding 🙏🏻
